### ISSUE TITLE
p4 only plays bad if *both* skill level AND elos are adjusted

### ISSUE THREAD

**Issue by rofl0r** [Adam-Kulju/Patricia#121]
using only one of the combinations UCI_LimitStrength+UCI_Elo value 500 or Skill_Level value 1 doesn't result in 500 elo play, in fact it beats most engines i tested it against including toga III, and almost produced a draw against stockfish 11.
to get it to actually play at that level all mentioned options need to be enabled. from my reading of the release notes, either way should work. though maybe the Skill_Level lever should have a default setting of "uncapped" (0?) or something like that.

**Comment by rofl0r** [comment]
btw i wonder what depth is required to get the desired elo strength. i've read somewhere that human grandmasters can dive to max depth 12, so i figure for elo 500 one would need something like "go depth 2" ?

**Comment by Adam-Kulju** [comment]
Yeah, something I overlooked is that if you set "Skill_Level" to 10 but leave "UCI_Elo" untouched, the GUI sets both and often the UCI_Elo value is the one that winds up getting used for the skill level. This will be fixed in future updates.

Patricia doesn't do limited depth or nodes searches because `go depth 1` is still stronger than 500, and hangs thoughtless mates way more often than it should.

**Comment by rofl0r** [comment]
thanks for your answer, though the intent of the question was to find the depth level which should be used to accompany the desired elo setting. i figure even at 500 elo, if you let the engine search till it hits depth 30, it will still pick a pretty good move that's way above the desired level.

> go depth 1 is still stronger than 500

does that mean it's actually depth 1 + quiescence search ?

**Comment by Adam-Kulju** [comment]
yes, correct

**Comment by rofl0r** [comment]
> though maybe the Skill_Level lever should have a default setting of "uncapped" (0?) or something like that.

on second thought, if this was my engine, i'd just remove the Skill_Level setting completely. those that would set it, would most likely look up which setting corresponds to the desired elo level, so they could just set the elo setting to begin with. and unlike the Skill_Level setting, it has a dedicated and well-known switch to toggle it on/off.
